fees/fluid/fees.ts

```diff
@@ -123,16 +123,21 @@ export const getVaultsT1Resolver = async (api: ChainApi) => {
   }
 
   return {
-    getAllVaultsAddresses: async () => api.call({ target: address, abi: abi.getAllVaultsAddresses }),
-    getVaultEntireData: async (vaults: string []) => {
-      return api.multiCall({ calls: vaults.map((vault) => ({ target: address, params: [vault] })), abi: abi.getVaultEntireData });
-    }
+    getAllVaultsAddresses: async () => { 
+      let vaults = await api.call({ target: address, abi: abi.getAllVaultsAddresses });
```

Comment:
this replaces the removed permitFailure logic. As mentioned there is a timespan on Arbitrum where reverts would happen without adding such logic.